frontend/src/components/sightings/sightings.tsx

```diff
@@ -1,26 +1,45 @@
 import { useEffect, useState } from "react";
 import {
-  fetchSightings,
-  SightingResponse,
+  fetchSightings as fetchInternalSightings,
+  SightingResponse as internalSightingResponse,
 } from "../../clients/internalApiClient";
-import { SightingCard } from "../sightingCard/sightingCard";
+import {
+  fetchSightings as fetchExternalSightings,
+  SightingResponse as externalSightingResponse,
+} from "../../clients/externalApiClient";
+import { SightingCard as SightingCard } from "../sightingCard/sightingCard";
 
 export const Sightings: React.FunctionComponent = () => {
-  const [sightings, setSightings] = useState<SightingResponse[]>();
+  const [internalSightings, setInternalSightings] =
+    useState<internalSightingResponse[]>();
+  const [externalSightings, setExternalSightings] =
+    useState<externalSightingResponse[]>();
 
   useEffect(() => {
-    fetchSightings().then((response) => setSightings(response.sightings));
+    fetchInternalSightings().then((response) =>
+      setInternalSightings(response.sightings)
+    );
+  }, []);
+
+  useEffect(() => {
+    fetchExternalSightings().then((response) =>
+      setExternalSightings(response.sightings)
+    );
   }, []);
 
   return (
     <section>
       <h1>Sightings</h1>
-      {sightings &&
-        sightings.map((sighting) => {
+      {internalSightings &&
+        internalSightings.map((sighting) => {
           if (sighting.isApproved) {
             return <SightingCard sighting={sighting} key={sighting.id} />;
           }
         })}
+      {externalSightings &&
+        externalSightings.map((sighting) => {
+          return <SightingCard sighting={sighting} key={sighting.id} />;
+        })}
```

Comment:
They shouldn't appear one after the other, they should be interleaved and ordered by date! Might be a bit tricky I'm afraid